app/src/main/res/layout/view_labeled_address_item.xml

```diff
@@ -4,65 +4,28 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="68dp"
+    android:layout_height="wrap_content"
     tools:parentTag="androidx.constraintlayout.widget.ConstraintLayout">
 
     <TextView
-        android:id="@+id/label"
-        style="@style/TextDark.Bold"
+        android:id="@+id/name"
```

Comment:
Why is this called `name` and not `label` as the file name suggests? And in `view_named_address_item.xml` the id is called `label` instead of `name`.